src/main/java/network/warzone/scaffold/commands/ScaffoldCommands.java

```diff
@@ -238,7 +238,7 @@ public static void upload(CommandContext cmd, CommandSender sender) {
 
             sender.sendMessage(ChatColor.YELLOW + "Uploading world...");
             try {
-                HttpResponse<String> response = Unirest.post("https://transfer.sh/").header("Max-Downloads", "1").header("Max-Days", "3").field("upload-file", zip).asString();
+                HttpResponse<String> response = Unirest.post("https://transfer.sh/").header("Max-Downloads", Scaffold.get().getConfig().getString("export.maxdownloads")).header("Max-Days", Scaffold.get().getConfig().getString("export.maxdays")).field("upload-file", zip).asString();
```

Comment:
To reduce the width of this line, store the config retrieved from `Scaffold.get().getConfig()` into a local variable and provide a default value for the config paths you are requesting, as such: `config.getString("export.maxdownloads", 1)`